tools/perf/brave_page_sets/brave_system_health/system_health_stories.py

```diff
@@ -47,6 +47,29 @@ def GenerateStoryDescription(cls):
     return 'Load %s' % cls.URL
 
 
+class _BraveMultiTabLoadingStory(_BraveLoadingStory):
```

Comment:
It's formally a system_health story, but in fact it's can't work with `system_health.common*`
Also I'm not sure about Android. 

Please add a assert / comment that it's used only for memory.

Reply:
For now, I have added the comment in the class description itself, but I couldn't retrieve the benchmark names to add assert :thinking:, kindly let me know if you have any ideas on this.

I haven't seen any platform specific differences so far. Could you help me with the testing on Android to verify?

Reply:
Ok, adding the comment is acceptable.

Reply:
The check android we need to fix https://github.com/brave/brave-core/pull/25255#issuecomment-2320520016

Reply:
Done :+1: